Added comment to GetCredentials

gulyasm opened the pull request:
# What is this PR about?
The `GetCredentials` function had no documentation making it harder to understand it's place in the authentication process.

# How this PR solves it?
This PR adds documentation to the function using the same terminology as in the Twitter documentation. 

# How to test it?
Only documentation, no need to run anything.

# Risks
Only documentation, but documentation can mislead the user if not precise or clear.

gulyasm commented:
@ChimeraCoder Can I do anything to help this PR?

muesli commented:
Could you fix the typo "credentianls" please? I think the description / comment is still a bit vague: "GetCredentials implements the third part of the OAuth handshake."

Could you also rebase it to master?

@ChimeraCoder: Anything you object to otherwise?

gulyasm commented:
Fixed the typo, rebased to master. 

> think the description / comment is still a bit vague:

@muesli It's vague because it's not obvius where you should use it (which part of the handshake) or somehting else?
